man/man3/fi_peer.3

```diff
@@ -454,7 +454,7 @@ struct fi_peer_rx_entry {
     struct fi_peer_rx_entry *prev;
     struct fi_peer_srx *srx;
     fi_addr_t addr;
-    size_t size;
+    size_t msg_size;
```

Comment:
Man page changes should be made to the markdown files. The actual man page is updated automatically.

Reply:
Oops! I think VS Code opened the wrong file for me... Updated. Thanks for catching!